@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardCpuProvisionning.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <UiCard>
+    <UiTitle type="h4">{{ $t("cpu-provisioning") }}</UiTitle>
+    <div v-if="isReady" class="progress-item">
+      <UiProgressBar :value="nVCpuInUse" :max-value="nPCpu * SCALE" />
+      <UiScaleProgressBar :scale="SCALE" />
+      <UiLegendProgressBar>
+        <template #label>{{ $t("vcpus") }}</template>
+        <template #value>{{ value }}%</template>
+      </UiLegendProgressBar>
+      <div class="footer">
+        <div class="footer-card">
+          <p>{{ $t("vcpus-used") }}</p>
+          <span class="footer-value">{{ nVCpuInUse }}</span>
+        </div>
+        <div class="footer-card">
+          <p>{{ $t("total-cpus") }}</p>
+          <span class="footer-value">{{ nPCpu }}</span>
+        </div>
+      </div>
+    </div>
```

Comment:
It may be better to create a separate component to display a footer card, as it is used in multiple cards.